client/header/activity-panel/panels/inbox/placeholder.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { Component } from '@wordpress/element';
+import PropTypes from 'prop-types';
+
+class InboxNotePlaceholder extends Component {
+	render() {
+		const { className } = this.props;
+		const cardClassName = classnames(
+			'woocommerce-inbox-message',
+			'is-placeholder',
+			className
+		);
```

Comment:
I think this can just be a string concatenation instead using `classnames` since there isn't any logic/complexity here.

Reply:
Yes, the method `classnames` is unnecessary. I removed it in the commit c571756